content/docs/examples/multicluster/icp/index.md

```diff
@@ -0,0 +1,219 @@
+---
+title: Enabling Multicluster on IBM Cloud Private
+description: Example multicluster IBM Cloud Private install of Istio.
+weight: 70
+keywords: [kubernetes,multicluster]
+---
+
+This example demonstrates how to use Istio' multicluster feature to join two
+[IBM Cloud Private](https://www.ibm.com/cloud/private) clusters together,
+using the [kubernetes multicluster installation instructions](/docs/setup/kubernetes/multicluster-install/).
+
+## Create the IBM Cloud Private Clusters
+
+1.  You can follow steps [here](https://www.ibm.com/support/knowledgecenter/en/SSBS6K_2.1.0.3/installing/installing.html)
+    to install two IBM Cloud Private clusters.
+    __NOTE__: Make sure individual cluster Pod CIDR ranges and service CIDR ranges must be unique across the multicluster
+    environment and may not overlap. This can be configured by `network_cidr` and `service_cluster_ip_range` in `cluster/config.yaml`.
+    
+    {{< text bash >}}
+    ## Network in IPv4 CIDR format
+    network_cidr: 10.1.0.0/16
+
+    ## Kubernetes Settings
+    service_cluster_ip_range: 10.0.0.1/24
+    {{< /text >}}
+
+1.  After IBM Cloud Private cluster install finsihed, validate `kubectl` access to each cluster. Suppose we have
+    two clusters: cluster-1 and cluster-2.
+
+    1.  Follow steps [here](https://www.ibm.com/support/knowledgecenter/SSBS6K_2.1.0.3/manage_cluster/cfc_cli.html)
+        to config `kubectl` for cluster-1, and run following command to check cluster status.
+
+        {{< text bash >}}
+        $ kubectl get nodes
+        $ kubectl get pods --all-namespaces
+        {{< /text >}}
+
+    1.  Use same steps as cluster-1 above to check cluster-2.
+    
+## Config Pod Communication Cross IBM Cloud Private Clusters
+
+IBM Cloud Private is using Calico Node-to-Node Mesh by default to manage container networks. The BGP Client
+on each node distributes the IP router information to all nodes.
+
+In order to make sure pods can communicate across different clusters, you need to configure IP routers on
+all nodes in the cluster. You need to add IP routers in cluster-2 to cluster-1, and add IP routers in
```

Comment:
Might break these into steps such as

1. Add IP routers from `cluster-1` to `cluster-2`
1. Add IP routers from `cluster-2` to `cluster-1`

Reply:
Done